swagger/swagger.json

```diff
@@ -1332,6 +1366,23 @@
       },
       "description" : "A POST link related to a payment"
     },
+    "PrefilledCardholderDetails" : {
+      "type" : "object",
+      "properties" : {
+        "cardholder_name" : {
+          "type" : "string",
+          "example" : "J. Bogs",
+          "description" : "prefilled cardholder name",
+          "minLength" : 0,
+          "maxLength" : 255
+        },
+        "billing_address" : {
+          "description" : "prefilled billing address",
+          "readOnly" : true,
+          "$ref" : "#/definitions/Address"
+        }
+      }
+    },
```

Comment:
I'm confused why some of these things are being added as part of this PR?

Reply:
For some odd reason the swagger file changed when I made my code - I must of accidentally committed all. At any rate I've rebased and the swagger file has been changed since then.